apps/plato_single_variant/Readme.md

```diff
@@ -24,26 +21,26 @@ There are two input options required to run the app:
     | 4 |   4 |   1   |   5.3    |
 
 Other optional inputs:
-- Continuous Covariates: A tab-separated file with columns: `FID`,`IID`,`Cov1`,`Cov2`,`....`
-Here values of the covariate should be numeric and quantitative.
-- Categorical Covariates: A tab-separated file with columns: `FID`,`IID`,`Cat_Cov1`,`Cat_Cov2`,`....`.
-Here entry of non-numeric classes are allowed and will automatically generate a dummy encoding.
-- Sample File: User-specified list of sample ID's to include in the analysis. The file format should be a space/tab-delimited text file with family IDs (`FID`) in the first column and individual IDs (`IID`) in the second column.
-- SNP file: User-specified list of SNPs to include in the analysis. The file format should be either one `RSID` per line or positions in rage format (`chr`,`start`,`stop`,`label`)
+  - Continuous Covariates: A tab-separated file with columns: `FID`,`IID`,`Cov1`,`Cov2`,`....`
+    - Here values of the covariate should be numerical continuous.
+  - Categorical Covariates: A tab-separated file with columns: `FID`,`IID`,`Cat_Cov1`,`Cat_Cov2`,`....`.
+    - Here entry of non-numeric classes are allowed and will automatically generate a dummy encoding.
```

Comment:
Here, entry of non-numeric classes are allowed and will automatically generate a dummy encoding.